src/cmd/check.go

```diff
@@ -27,9 +28,12 @@ func Check(cfg *MainConfig) (int, error) {
 	var args cmdlineArguments
 	bindFlags(config, ruleSets, &args)
 	flag.Parse()
+
 	if args.disableCache {
 		config.CacheDir = ""
 	}
+	config.PhpExtensions = strings.Split(args.phpExtensionsArg, ",")
```

Comment:
This is not the best place to initialize the config.
linterRunner does the args->config assignments.